pkg/auth/server.go

```diff
@@ -56,6 +56,8 @@ func NewServer(ctx context.Context, cfg *rest.Config, wContext *wrangler.Context
 		return nil, err
 	}
 
+	sc.Wrangler = wContext
```

Comment:
I have nothing against it, but is there a reason you moved setting the Wrangler context earlier here?